src/rust/src/lib.rs

```diff
@@ -109,6 +109,9 @@
 //! ```
 //!
 
+#[macro_use]
+extern crate log;
```

Comment:
If it would be less controversial, I'd be happy to remove this and do `logger::trace`, `logger:debug`, etc. at all the call sites. 

This is just the way that I typically do it, so I thought I'd start there.